docs/app/pages/components/components-sections/icon/icon/snippets/appModule.ts

```diff
@@ -0,0 +1,17 @@
+import { IconModule } from '@ux-aspects/ux-aspects';
+import { NgModule } from '@angular/core';
+
+@NgModule({
+    imports: [
+        IconModule.forRoot({
+            icons: [
+                { name: '3d', icon: 'qtm-icon-ci', iconset: 'qtm-font-icon' , size: '14px'},
+                { name: 'alert', icon: 'qtm-icon-alarm', iconset: 'qtm-font-icon', size: '16px' }
+            ]
+        })
+    ],
+})
+
```

Comment:
Can we remove this empty line between the decorator and class

Reply:
Removed